res/gamecontrollerdb_2016.txt

```diff
@@ -0,0 +1,1937 @@
+# Game Controller DB for SDL in 2.0.16 format
```

Comment:
What's the purpose of this file? Seems like it might fall out of date quickly. Can't we detect these things sort-of automatically?

Reply:
This is a leftover from SDL. Will remove it